File: test/fixtures/php-guzzle/put_with_T_option.php
<?php
require 'vendor/autoload.php';

use GuzzleHttp\Client;
use GuzzleHttp\Psr7;

$client = new Client();

$response = $client->put('http://localhost:28139/twitter/_mapping/user?pretty', [
    'headers' => [
        'Content-Type' => 'application/json'
    ],
    'body' => Psr7\Utils::tryFopen('my_file.txt', 'r')
]);

File: test/fixtures/ruby-httparty/put_with_T_option.rb
require 'httparty'

url = 'http://localhost:28139/twitter/_mapping/user?pretty'
headers = {
  'Content-Type': 'application/json',
}
body = File.read('my_file.txt')
res = HTTParty.put(url, headers: headers, body: body)
